client/src/App.js

```diff
@@ -12,11 +13,22 @@ class App extends Component {
 
   async componentDidMount() {
     const tags = await getTags()
-    this.setState({tags})
+    const reportDataByTag = {}
+    for (const tagObject of tags) {
+      const reportData = await getReportsByTag(tagObject.tag)
```

Comment:
`tagObject` and `reportData` are named so because I wanted to avoid writing `tag.tag` and `report.report`. But any other naming suggestion is welcome.